non-free.md

```diff
@@ -7,7 +7,7 @@
 - [66Analytics](https://66analytics.com/) `⊘ Proprietary` - 66Analytics is a self-hosted, friendly, all-in-one web analytics tool. Lightweight tracking, session replays, heatmaps, user journeys & more. Can be used for personal reasons & can be white labeled to start a SAAS. ([Demo](https://66analytics.com/demo)) `PHP/MySQL`
 - [userTrack](https://www.usertrack.net/) `⊘ Proprietary` - userTrack is a web analytics platform with heatmaps, session recordings and powerful user segmentation feature. Updated very regularly. ([Demo](https://dashboard.usertrack.net/server/demoLogin.php)) `PHP/MySQL/ReactJS`
 - [UXLens](https://uxlens.com/) `⊘ Proprietary` - UXLens is a website visitor recording software meant for identifying UI issues and fix them to improve user experience. Formerly known as SessionRecord ([Demo](https://console.uxlens.com/test)) `Docker Nodejs`
-
+- [ElasticSearch](https://www.elastic.co/elasticsearch/) - `⊘ Proprietary` - A distributed, RESTful search and analytics engine. ([Source Code](https://github.com/elastic/elasticsearch)) `Java` 
```

Comment:
- Please keep the empty line
- Please add `SSPL-1.0` in the same was as https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/non-free.md#time-tracking

Reply:
Should be all good now. I added a space between automation and elastic. Unless you want a empty space between UXLens and ElasticSearch.